### DISCUSSION TITLE
getUser(token) function location. From docs.

### DISCUSSION THREAD

**Discussion by HenryOwenz** [apollographql/apollo-server#7354] [Q&A] [answered]
I have a silly question. Where is the getUser() function call coming from in this documentation? (https://github.com/apollographql/apollo-server/blob/main/docs/source/security/authentication.mdx) Is it just example? And if so what would be the getUser functionality?

**Comment by trevor-scheer** [comment, marked as answer]
@HenryOwenz it is just example, yes. Here, `getUser` is a supposed function that uses a token sent via headers to look up a `user` object with some potentially useful details on it needed by your resolvers via the `context` object.

**Reply by HenryOwenz** [reply]
Got it. Thank you for the response!
